dapper/mods/VL20/__init__.py

```diff
@@ -1,10 +1,12 @@
 """
-One scale modified model of Lorenz96 which added one thermodynamic variable.
+One scale modified model of Lorenz96 with an added thermodynamic variable.
```

Comment:
```suggestion
Single-scale Lorenz-96 with an added thermodynamic component.
```
Would that be correct?
Also, no newline at top.

Reply:
It is correct and better. Thanks for the suggestion.